sql/moz-fx-data-marketing-prod/acoustic/suppression_list_v1/query.sql

```diff
@@ -0,0 +1,36 @@
+WITH base_suppression_lists AS (
+  SELECT
+    email
+  FROM
+    `moz-fx-data-bq-fivetran.acoustic_sftp.suppression_export_v_1`
+  UNION ALL
+  SELECT
+    email
+  -- historic upload
+  FROM
+    `moz-fx-data-marketing-prod.acoustic.2024-03-20_main_suppression_list`
+),
+suppression_list AS (
+  SELECT DISTINCT
```

Comment:
We could avoid this extra step by just using UNION DISTINCT in the step before?

Reply:
TIL
updated it